### ISSUE TITLE
Crash when returning to homescreen

### ISSUE THREAD

**Issue by eengnr** [MM2-0/Kvaesitso#436]
## Description

The latest version 1.26.0 crashes randomly when returning to the home screen from an app.

## Strack trace

```
java.lang.IllegalStateException: Offset is unspecified
	at androidx.compose.ui.geometry.Offset.getX-impl(Offset.kt:29)
	at androidx.compose.ui.node.NodeCoordinator.fromParentPosition-MK-Hz9U(NodeCoordinator.kt:3)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:19)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
[69 lines truncated]
```

## Device info
APP.VERSION : 1.26.0<br>APP.VERSIONCODE : 2023062300<br>APP.SIGNATURE : rx1fSnL7r5/OMoFC0e1KPqTndXQ=<br>LAUNCHER.APP : android<br>TIMEZONE : Europe/Berlin<br>VERSION.RELEASE : 13<br>VERSION.INCREMENTAL : 9891397<br>VERSION.SDK.NUMBER : 33<br>BOARD : panther<br>BOOTLOADER : cloudripper-1.0-9618366<br>BRAND : google<br>CPU_ABI : arm64-v8a<br>CPU_ABI2 : <br>DISPLAY : TQ2A.230505.002<br>FINGERPRINT : google/panther/panther:13/TQ2A.230505.002/9891397:user/release-keys<br>HARDWARE : panther<br>HOST : abfarm-release-rbe-64-2004-0030<br>ID : TQ2A.230505.002<br>MANUFACTURER : Google<br>MODEL : Pixel 7<br>PRODUCT : panther<br>TAGS : release-keys<br>TIME : 1680732913000<br>TYPE : user

**Comment by Sir-Photch** [comment]
Also getting this.

```
type: crash
osVersion: google/cheetah/cheetah:13/TQ3A.230605.012/2023061402:user/release-keys
package: de.mm20.launcher2.release:2023062300
process: de.mm20.launcher2.release
processUptime: 18110 + 247 ms

java.lang.IllegalStateException: Offset is unspecified
	at androidx.compose.ui.geometry.Offset.getX-impl(Offset.kt:29)
	at androidx.compose.ui.node.NodeCoordinator.fromParentPosition-MK-Hz9U(NodeCoordinator.kt:3)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:19)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.ancestorToLocal-R5De75A(NodeCoordinator.kt:15)
	at androidx.compose.ui.node.NodeCoordinator.localPositionOf-R5De75A(NodeCoordinator.kt:80)
	at androidx.compose.ui.node.NodeCoordinator.windowToLocal-MK-Hz9U(NodeCoordinator.kt:29)
	at androidx.compose.foundation.text.selection.SelectionManagerKt.visibleBounds(SelectionManager.kt:13)
	at androidx.compose.foundation.text.CoreTextFieldKt$CoreTextField$onPositionedModifier$1.invoke(CoreTextField.kt:202)
	at androidx.compose.ui.layout.OnGloballyPositionedNode.onGloballyPositioned(OnGloballyPositionedModifier.kt:3)
	at androidx.compose.ui.node.OnPositionedDispatcher.dispatchHierarchy(OnPositionedDispatcher.kt:61)
	at androidx.compose.ui.node.OnPositionedDispatcher.dispatchHierarchy(OnPositionedDispatcher.kt:159)
	at androidx.compose.ui.node.OnPositionedDispatcher.dispatchHierarchy(OnPositionedDispatcher.kt:159)
	at androidx.compose.ui.node.OnPositionedDispatcher.dispatchHierarchy(OnPositionedDispatcher.kt:159)
	at androidx.compose.ui.node.OnPositionedDispatcher.dispatchHierarchy(OnPositionedDispatcher.kt:159)
	at androidx.compose.ui.node.OnPositionedDispatcher.dispatchHierarchy(OnPositionedDispatcher.kt:159)
	at androidx.compose.ui.node.OnPositionedDispatcher.dispatchHierarchy(OnPositionedDispatcher.kt:159)
	at androidx.compose.ui.node.OnPositionedDispatcher.dispatchHierarchy(OnPositionedDispatcher.kt:159)
	at androidx.compose.ui.node.MeasureAndLayoutDelegate.dispatchOnPositionedCallbacks(MeasureAndLayoutDelegate.kt:103)
	at androidx.compose.ui.platform.AndroidComposeView.updatePositionCacheAndDispatch(AndroidComposeView.android.kt:58)
	at androidx.compose.ui.platform.AndroidComposeView.onLayout(AndroidComposeView.android.kt:11)
	at android.view.View.layout(View.java:23798)
	at android.view.ViewGroup.layout(ViewGroup.java:6413)
	at androidx.compose.ui.platform.AbstractComposeView.internalOnLayout$ui_release(ComposeView.android.kt:28)
	at androidx.compose.ui.platform.AbstractComposeView.onLayout(ComposeView.android.kt:7)
	at android.view.View.layout(View.java:23798)
	at android.view.ViewGroup.layout(ViewGroup.java:6413)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
	at android.view.View.layout(View.java:23798)
	at android.view.ViewGroup.layout(ViewGroup.java:6413)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1891)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1729)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1638)
	at android.view.View.layout(View.java:23798)
	at android.view.ViewGroup.layout(ViewGroup.java:6413)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
	at android.view.View.layout(View.java:23798)
	at android.view.ViewGroup.layout(ViewGroup.java:6413)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1891)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1729)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1638)
	at android.view.View.layout(View.java:23798)
	at android.view.ViewGroup.layout(ViewGroup.java:6413)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
	at com.android.internal.policy.DecorView.onLayout(DecorView.java:799)
	at android.view.View.layout(View.java:23798)
	at android.view.ViewGroup.layout(ViewGroup.java:6413)
	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:4114)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3455)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:2367)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:9293)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1231)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1239)
	at android.view.Choreographer.doCallbacks(Choreographer.java:899)
	at android.view.Choreographer.doFrame(Choreographer.java:832)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1214)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7937)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)

```

**Comment by MM2-0** [comment]
Fixed in 1.26.1
